package.json

```diff
@@ -565,6 +572,11 @@
             ]
         },
         "commands": [
+            {
+                "command": "extension.vsKubernetesDescribe.Refresh",
+                "title": "Refresh",
```

Comment:
This might be confusing because it specifically refreshes the `Describe` window.  It is probably okay if we set up a `commandPalette` entry so that it's only shown when a describe window is active.

Reply:
I always am unclear about how to do this, suggestions welcome...

Reply:
Yeah, I haven't found a great solution.  You can either:

* Make the title very explicit: `Refresh Active Description`
* Add a `contributes/menus/commandPalette` entry with a `when` clause of `context == describePanel`  (or something like that), so that the `Refresh` entry doesn't appear in the palette unless the user is looking at a description
* Use a category: `"category": "Kubernetes Describe", "title": "Refresh"` - when seen in the command palette it will be qualified by the category, disambiguating it from any other Refresh commands